rockcraft.yaml

```diff
@@ -0,0 +1,50 @@
+name: charmed-karapace
+summary: Charmed Karapace ROCK
+description: |
+  This is an OCI image that bundles Karapace binaries together with other
+  tools of its ecosystem in order to be used in Charmed Operators, providing
+  an automated and seamless experience to deploy, operate, manage and monitor
+  Karapace on K8s cluster.
+  It is an open source, end-to-end, production ready data platform on top of
+  cloud native technologies.
+
+license: Apache-2.0
+version: '3.12.0'
+base: ubuntu@22.04
+platforms:
+  amd64:
+
+run_user: _daemon_
+
+services:
+  karapace:
+    command: karapace /etc/karapace/karapace.config.json
```

Comment:
**niptick** maybe we could feed the config file using an env variable, such that one can customize this?

Reply:
Leaving this out for a follow up. If we want to add this, the auth file should also be added along the options to create the users for it.